Makefile

```diff
@@ -11,146 +11,271 @@
 # EXTERNAL VARIABLES
 # ======================
 
-REPOSITORY :=
-PREFIX :=
-TARGET := docker
+# The repository where the image is going to be hosted.
+# eg, ghcr.io/canonical [To be passed when you 'make' the recipe]
```

Comment:
**praise** Huge praise for providing these comments, that helps understand the Makefile!!!!